raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -37,49 +36,56 @@
     private static final String QUORUM_PREFIX = "quorum.";
```

Comment:
We need the `controller` prefix that comes from KIP-631. Also if you could change raft/README to reflect the new names, that would be helpful.

Reply:
Done.